shared.go

```diff
@@ -118,6 +119,43 @@ type Value struct {
 	Name string `json:"name"`
 }
 
+// Field is generic Name/Value struct applied to a few places.
+type Field struct {
```

Comment:
It might make sense to call this `OutputField` since it will never be an input.

Reply:
Right now, this is both input and output, I can either split them in `OutputField` and `InputField`, and keep 2 different `IndexerInput` and `IndexerOutput`, or have just one `Indexer` and one `Field`. Surely now it's inconsistent and must be changed.

Reply:
I didn't realize it was both. nbd then.